scripts/generate

```diff
@@ -191,7 +191,9 @@ def('substitutions',
                           K(el('span', 'arrow', htmlEncode('~>')))],
                          [contains(_, ap([I, htmlEncode], ['->'])),
                           K(el('span', 'arrow',
-                               el('span', 'arrow-hyphen', htmlEncode('-')) +
+                               //  Use NON-BREAKING HYPHEN so arrow does
+                               //  not break because of line wrapping
+                               el('span', 'arrow-hyphen', '&#x2011;') +
```

Comment:
I'm in favour of using an HTML entity for exactly this reason.

I'd like to see this issue addressed at its source: [Transcribe][1]. That way the Sanctuary readme (and all other Transcribe-generated readmes) will get the improved line wrapping, and the change will flow through to the Sanctuary website (which is created from the project's readme).

Specifically, I think we could change `' -> '` to `' \u2011> '` in [__bin/transcribe__][1]. Would you like to open a pull request for this change?


[1]: https://github.com/plaid/transcribe/blob/v0.5.0/bin/transcribe#L34